### PR TITLE
XCode 6 - no more GC

### PR DESCRIPTION
**Pull request by daicoden**
know idea

So it works again on modern macs... :-D

### CODE REVIEW

**Comment by daicoden** [comment]
Seems to be working with things integration... haven't tested anything else.

**Comment by daicoden** [comment]
cc @ecin it works again! the only pomodoro that's worth anything.

File: externalFw/OAuthConsumer/OADataFetcher.m [hunk 1]
```diff
@@ -40,26 +40,33 @@ - (void)fetchDataWithRequest:(OAMutableURLRequest *)aRequest
     didFailSelector = failSelector;
     
     [request prepare];
+    NSURLResponse *localResponse;
```
Comment (daicoden) [inline externalFw/OAuthConsumer/OADataFetcher.m]: This is probably the ugliest stuff I did manually... but not sure a better way.

**Comment by abencz** [comment]
Seems like pomodoro is becoming abandonware :(

Gonna try this fix later today.

**Comment by daicoden** [comment]
Any luck @abencz ?

This pull works for a few iteration, but the app will crash if the computer goes to sleep and you have to restart it.  I'm sure somethings getting collected that shouldn't be.

**Comment by abencz** [comment]
Haven't seen this since I use Pomodoro on my work mac that's basically docked & running all the time. I'll give it a shot at home.

**Comment by abencz** [comment]
Found a crash with the break timer. It's getting deallocated when it expires and then there's this check in `PomodoroController.m` when starting a new pomodoro:

        if ([longBreakCheckerTimer isValid]) {
            [longBreakCheckerTimer invalidate];
            longBreakCheckerTimer = nil;
        }

Openend [a PR](https://github.com/daicoden/pomodoro/pull/1) against your fork.

Going to keep using it for a few days and see what happens.

**Comment by abencz** [comment]
By the way, I get this weird graphical glitch when first launching the app - some of the timer is cut off:

pomodoro

Do you see the same thing?

**Comment by ryan-p-randall** [comment]
@abencz I'm running a much older version of Pomodoro & it's had that graphical glitch the whole time I've been using it. I doubt it's related to whatever changes you all are making. 

Also, thanks for working on this! I'd love to get it integrated with the Notification Center on my Yosemite machine.

**Comment by abencz** [comment]
I've had it for a while too, seems like it goes away after you start a pomodoro so I'm not overly concerned.

It's working pretty well otherwise. It does show up in notification center but not all notifications show up as alerts/banners.

**Comment by daicoden** [comment]
I'm closing this in favor of #312

**Pull request closed without merging**